### ISSUE TITLE
Student labels

### ISSUE THREAD

**Issue by MindImpMatt** [maltyeva/mindimp-pg#108]
Would it be possible to label students by their (most recent) academic term? I suppose this goes along with the active/inactive discussion, but would be a bit more about keeping track of more current students. It'd be especially helpful once we get a larger student population.

**Comment by maltyeva** [comment]
Yes, where are we labeling the students? I added a label to their profile, but could add it somewhere else as well.

**Comment by maltyeva** [comment]
also, advisor code is done, so will be pushing the changes today.

**Comment by MindImpMatt** [comment]
What about after their name on the checkbox lists for adding students to
class etc?

On Thu, Apr 30, 2015 at 10:22 AM, maltyeva <notifications@github.com> wrote:

> Yes, where are we labeling the students? I added a label to their profile,
> but could add it somewhere else as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/108#issuecomment-97635801>.
>



-- 
Matt Steyl | 荀慕天
Program Manager | MindImp 思脉
http://www.mindimp.com.cn/
https://www.mindimp.com/

**Comment by MindImpMatt** [comment]
Would that be ok? Or too cluttered?

On Thu, Apr 30, 2015 at 10:24 AM, Matt Steyl <matt@mindimp.com> wrote:

> What about after their name on the checkbox lists for adding students to
> class etc?
>
> On Thu, Apr 30, 2015 at 10:22 AM, maltyeva <notifications@github.com>
> wrote:
>
>> Yes, where are we labeling the students? I added a label to their
>> profile, but could add it somewhere else as well.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/maltyeva/mindimp-pg/issues/108#issuecomment-97635801>
>> .
>>
>
>
>
> --
> Matt Steyl | 荀慕天
> Program Manager | MindImp 思脉
> http://www.mindimp.com.cn/
> https://www.mindimp.com/
>
>


-- 
Matt Steyl | 荀慕天
Program Manager | MindImp 思脉
http://www.mindimp.com.cn/
https://www.mindimp.com/

**Comment by maltyeva** [comment]
ohh... hmm, I can definitely do that, but the thing is that I was basing the term labels on whether they have taken a course in that terms. There are a few ways we can go about this: 


1. Associate a student with a specific term so that they can only take classes for it (good as a long-term solution once we have students sorted by active/inactive
2. Also, or instead, have the students in the list of students added to class be sorted by most recent. 

Would either of those be helpful?

**Comment by MindImpMatt** [comment]
Either of those would be great! The first especially would be helpful to
filter out extra names on the checkbox lists.

On Thu, Apr 30, 2015 at 10:37 AM, maltyeva <notifications@github.com> wrote:

> ohh... hmm, I can definitely do that, but the thing is that I was basing
> the term labels on whether they have taken a course in that terms. There
> are a few ways we can go about this:
>
>    1. Associate a student with a specific term so that they can only take
>    classes for it (good as a long-term solution once we have students sorted
>    by active/inactive
>    2. Also, or instead, have the students in the list of students added
>    to class be sorted by most recent.
>
> Would either of those be helpful?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/108#issuecomment-97637322>.
>



-- 
Matt Steyl | 荀慕天
Program Manager | MindImp 思脉
http://www.mindimp.com.cn/
https://www.mindimp.com/

**Comment by maltyeva** [comment]
Ok, just sorted students by the date they joined the site! 

The thing about associated by a term is while that's doable, the class would need to be created first, since until the creation, it doesn't have a term. Would it be okay to create the class and then add the students to it in a separate action?

**Comment by maltyeva** [comment]
(this wouldn't be for labeling per se, but for filtering, i could do the labeling together, but it wouldn't filter)
